exporters/autoexport/exporter.go

```diff
@@ -102,16 +115,75 @@ func NewSpanExporter(ctx context.Context, opts ...Option) (trace.SpanExporter, e
 	if err != nil {
 		return nil, err
 	}
-	return config.fallbackExporter, nil
+	return config.fallbackSpanExporter, nil
+}
+
+// NewMetricReader returns a configured [go.opentelemetry.io/otel/sdk/metric.Reader]
+// defined using the environment variables described below.
+//
+// OTEL_METRICS_EXPORTER defines the metrics exporter; supported values:
+//   - "none" - "no operation" exporter
+//   - "otlp" (default) - OTLP exporter; see [go.opentelemetry.io/otel/exporters/otlp/otlptrace]
+//
+// OTEL_EXPORTER_OTLP_PROTOCOL defines OTLP exporter's transport protocol;
+// supported values:
+//   - "grpc" - protobuf-encoded data using gRPC wire format over HTTP/2 connection;
+//     see: [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc]
+//   - "http/protobuf" (default) -  protobuf-encoded data over HTTP connection;
+//     see: [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp]
+//
+// An error is returned if an environment value is set to an unhandled value.
+//
+// Use [RegisterMetricReader] to handle more values of OTEL_METRICS_EXPORTER.
+//
+// Use [WithFallbackMetricReader] option to change the returned exporter
+// when OTEL_TRACES_EXPORTER is unset or empty.
+//
+// Use [IsNoneSpanExporter] to check if the retured exporter is a "no operation" exporter.
+func NewMetricReader(ctx context.Context, opts ...Option) (metric.Reader, error) {
```

Comment:
Can we define a `MetricOption` instead of reusing existing `Option`.

We could also consider renaming `Option` to `SpanOption`, but this would have to be as a separate PR.

Reply:
I'll rename `Option` to `SpanOption` but leave `Option` as an alias for backwards compatibility. Later we can decide if/when we want to remove it.

Reply:
Given this is an unstable package, we could also just make the name change directly as long as we document it in the CHANGELOG. Otherwise, we should deprecate the alias now so we can remove it in the next release.

Reply:
I marked it as deprecated following the commenting convention. Do I also need to add it to the changelog?